libact/query_strategies/quire.py

```diff
@@ -12,21 +11,43 @@
 
 
 class QUIRE(QueryStrategy):
+    """Querying Informative and Representative Examples (QUIRE) query strategy
+
+
+    Parameters
+    ----------
+    lmbda: float, optional (default=1.0)
+
+
+    gamma: float, optional (default=1.0)
+
+
+    Attributes
+    ----------
```

Comment:
Don't we need docstrings?
I leave it blank for now just because I haven't have time to look into how to describe these parameters. hoping for @tungen to fill it in.